### ISSUE TITLE
Trouble building on Ubuntu 14.04

### ISSUE THREAD

**Issue by futurepr0n** [scottrice/Ice#325]
Hi I am having trouble on Ubuntu 14.04. 
At first the requirements.txt kept preventing me from moving forward until I changed it to pysteam>=1.0.0-b1 - after getting that to happen I was able begin running "python ice/"

When I this however I am getting the following error:

futurepr0n@futureMedia:~/Ice$ python ice/
/usr/lib/python2.7/dist-packages/pkg_resources.py:1031: UserWarning: /home/futurepr0n/.python-eggs is writable by group/others and vulnerable to attack when used with get_resource_filename. Consider a more secure location (set with .set_extraction_path or the PYTHON_EGG_CACHE environment variable).
  warnings.warn(msg, UserWarning)
=========== Starting Ice ===========
Detected Emulator: Dolphin
Detected Console: Nintendo Gamecube => Dolphin
=========== User: 2267067 ===========
[ERROR] An exception occurred while running Ice
Traceback (most recent call last):
  File "ice/runners/ice_engine.py", line 154, in run
    self.main(dry_run=dry_run)
  File "ice/runners/ice_engine.py", line 134, in main
    self.run_for_user(user_context, dry_run=dry_run)
  File "ice/runners/ice_engine.py", line 146, in run_for_user
    self._create_backup(user, dry_run=dry_run)
  File "ice/runners/ice_engine.py", line 164, in _create_backup
    shortcuts.write_shortcuts(backup_path, shortcuts.get_shortcuts(user))
  File "/usr/local/lib/python2.7/dist-packages/pysteam-1.0.0_beta2-py2.7.egg/pysteam/shortcuts.py", line 31, in write_shortcuts
    with open(path, "w") as f:
IOError: [Errno 2] No such file or directory: '/home/futurepr0n/.config/ice/Backups/2267067/config/shortcuts.20151228205645.vdf'

Close the window, or hit enter to exit...

I used to have the same error except it was slightly different path, because I was using pysteam-1.0.0-beta1 and then I decided I would install pysteam from the same repo and try with that.

**Comment by Andre-Oliveira** [comment]
To circumvent that problem create that file using the "touch" command.

**Comment by RobLoach** [comment]
I had to:

```
mkdir -p /home/futurepr0n/.config/ice/Backups/2267067/config
```

**Comment by scottrice** [comment]
I just fixed this issue with c05dd6c77f84d31f78b3a2ebaeb316df899b8167

**Comment by RobLoach** [comment]
:+1:
